### PR TITLE
feat: Google Analytics support

### LINKED ISSUE #52: feat: Google Analytics support
Add support for Google Analytics by adding the following tag to every page of the website, immediately after the `<head>` element (the `MEASUREMENT-ID` needs to be replaced of course):

```html
<!-- Google tag (gtag.js) -->
<script async src="https://www.googletagmanager.com/gtag/js?id=MEASUREMENT-ID"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'MEASUREMENT-ID');
</script>
```

You will need to find out how to do this in the proper Next JS way. Online resources will help, e.g., see [here](https://mariestarck.com/add-google-analytics-to-your-next-js-application-in-5-easy-steps/) and [here](https://nextjs.org/docs/messages/next-script-for-ga).

I will share the `MEASUREMENT-ID` privately, please contact me.

### PR DESCRIPTION
**Pull request by tanya17-05**
resolves #52 
to see logs use [google analytics debugger chrome extension](https://chrome.google.com/webstore/detail/google-analytics-debugger/jnkmfdileelhofjcijamephohjechhna/related?hl=en)

### CODE REVIEW

**Review by uniqueg — DISMISSED** [review]
Awesome! LGTM. Anything from your side, @git-anurag-hub? If not, I think you can go ahead and merge

**Review by git-anurag-hub — CHANGES_REQUESTED** [review]

File: components/CookieConsent.js [hunk 1]
```diff
@@ -1,5 +1,7 @@
 import { useEffect } from 'react';
 import 'vanilla-cookieconsent';
+import Cookies from 'js-cookie';
+import ReactGA from 'react-ga4';
```
[the remark this answers is not in the document]
Comment (tanya17-05) [reply components/CookieConsent.js:4]: That's true @git-anurag-hub, but I guess, we may run not able to track cookies as per user instructions. Script will be inserted in the webpage, regardless of user consent.

[the remark this answers is not in the document]
Comment (uniqueg) [reply components/CookieConsent.js:4]: And that's of course important. What do you say, @git-anurag-hub, do you know a better method that still regards the user's consent? If not, I guess you can resolve the issue and merge.

Thanks @tanya17-05

[the remark this answers is not in the document]
Comment (uniqueg) [reply components/CookieConsent.js:4]: @git-anurag-hub: Any update on this? Would be nice to get this in...

**Review by git-anurag-hub — CHANGES_REQUESTED** [review]

File: components/CookieConsent.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (git-anurag-hub) [reply components/CookieConsent.js:4]: Ahh, I actually did review this but forgot to hit submit so the review what pending from March 2nd. Will take this on priority!

**Pull request closed without merging**